build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -245,7 +245,7 @@ private static bool CheckIfAllBuildsHavePublished()
             nameof(PublishSharedFrameworkDebToDebianRepo),
             nameof(PublishHostFxrDebToDebianRepo),
             nameof(PublishSharedHostDebToDebianRepo))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, BuildPlatform.Debian)]
```

Comment:
Should `BuildPlatform.Debian` also be added to the `PublishInstallerFilesToAzure` method in this file?